api/src/main/java/org/pmiops/workbench/cohorts/CohortMaterializationService.java

```diff
@@ -223,6 +252,50 @@ private void addFilterOnConcepts(TableQuery tableQuery, Set<Long> conceptIds,
     }
   }
 
+  @VisibleForTesting
+  CdrQuery getCdrQuery(@Nullable CohortReview cohortReview, SearchRequest searchRequest,
+      @Nullable Set<Long> conceptIds, DataTableSpecification dataTableSpecification) {
+    CdrVersion cdrVersion = CdrVersionContext.getCdrVersion();
+    CdrQuery cdrQuery = new CdrQuery();
```

Comment:
nit: can use chained setters to initialize this

Reply:
Done.